### PR TITLE
Fixes for GCC11 and AL2022

### PR DESCRIPTION
**Pull request by lutkerd**
All builds passed and local AL2022 build worked.

### CODE REVIEW

File: make/lib/Awt2dLibraries.gmk [hunk 1]
```diff
@@ -433,7 +433,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBJAVAJPEG, \
     OPTIMIZATION := HIGHEST, \
     CFLAGS := $(CFLAGS_JDKLIB), \
     HEADERS_FROM_SRC := $(LIBJPEG_HEADERS_FROM_SRC), \
-    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value, \
+    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value array-bounds, \
```
Comment (benty-amzn) [inline make/lib/Awt2dLibraries.gmk]: Does this warning occur many times or would it make sense to disable it with pragmas in a few individual places?

Comment (mrserb) [reply make/lib/Awt2dLibraries.gmk]: I think it is fixed upstream here:
https://github.com/openjdk/jdk11u-dev/pull/700/files

Comment (lutkerd) [reply make/lib/Awt2dLibraries.gmk]: This is a backport from upstream, I don't want to make divergent changes.

Comment (lutkerd) [reply make/lib/Awt2dLibraries.gmk]: @mrserb Thanks! A few of these may be backported by Paul already, let me rebase on nightly and see. I don't want to wait for 11u-dev to get merged into 11u.

**Pull request closed without merging**